walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -25,22 +25,54 @@
 public final class FindMeetingQuery {
 
   /**
-   * Returns a Collection of TimeRange objects representing possible intervals of a certain meeting
-   * occurring, with all participants able to go and having a certain possible duration.
-   * @param  events  A Collection of Event objects formed before the creation of this meeting
-   * @param  request A MeetingRequest object representing the meeting data.
+   * Returns a Collection of TimeRange objects representing possible intervals of
+   * a certain meeting occurring, with all participants able to go and having a
+   * certain possible duration.
```

Comment:
Seems like the comment also needs to update for the optional attendees?

Reply:
Done.